pkg/api/vm/handler.go

```diff
@@ -1244,6 +1244,9 @@ func getClonedVMYamlFromSourceVM(newVMName string, sourceVM *kubevirtv1.VirtualM
 		},
 		Spec: *sourceVM.Spec.DeepCopy(),
 	}
+	if sourceReservedMemory := sourceVM.Annotations[util.AnnotationReservedMemory]; sourceReservedMemory != "" {
+		newVM.Annotations[util.AnnotationReservedMemory] = sourceReservedMemory
+	}
```

Comment:
let's define a constant array key and loop to add if they exist for better extendability, thanks.

Reply:
updated, add a slice var ([]string cannot be a const)